oncodash-app/index.html

```diff
@@ -18,7 +18,7 @@
   </style>
   <title>oncodash-app</title>
 </head>
-
+<h1>Hello Oncodash</h1>
```

Comment:
This should be inside the `<body>` tag :)

Reply:
Resolved here: https://github.com/oncodash/oncodash/pull/15/commits/90e125533519d6505a30500f7683b93f25a15205 :)